app/routes/home.tsx

```
@@ -1,13 +1,38 @@
+import ResumeCard from "~/components/ResumeCard";
+import { resumes } from "../../constants";
 import type { Route } from "./+types/home";
-import { Welcome } from "../welcome/welcome";
-
-export function meta({}: Route.MetaArgs) {
+import Navbar from "~/components/Navbar";
+import Footer from "~/components/Footer";
+export function meta({ }: Route.MetaArgs) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)